pygsti/circuits/gstcircuits.py

```diff
@@ -560,25 +567,38 @@ def add_to_plaquettes(pkey_dict, plaquette_dict, base_circuit, maxlen, germ, pow
                 add_to_plaquettes(pkey, plaquettes, empty_germ, maxLen, empty_germ, 1,
                                   allPossiblePairs, dscheck, missing_list)
                 unindexed.extend(filter_ds(lgst_list, dscheck, missing_lgst))  # overlap w/plaquettes ok (removed later)
-
+                #print(len(unindexed))
             #Typical case of germs repeated to maxLen using r_fn
             for ii, germ in enumerate(germs):
+                #print('-------------------Current Germ: ', germ)
+            
                 if germ == empty_germ: continue  # handled specially above
+                #print('Germ Length Limit', germ_length_limits.get(germ, 1e100))
                 if maxLen > germ_length_limits.get(germ, 1e100): continue
+                    
                 germ_power = truncFn(germ, maxLen)
+                #print('Current Germ: ', germ)
+                #print('Germ Power: ', germ_power)
                 power = len(germ_power) // len(germ)  # this *could* be the germ power
+                #print('Germ x Power: ', germ*power)
                 if germ_power != germ * power:
                     power = None  # Signals there is no well-defined power
-
+                #print('Power: ', power)
+                
+                if power==0 and len(germ) !=0:
+                    continue
```

Comment:
Been a while since we thought about this. I do recall something like this being the solution we settled on, and I don't see how it can introduce issues to default LSGST construction. If all tests pass, I say this is fine.